Archive/Examples/IfNormalization/Result.lean

```diff
@@ -92,11 +92,28 @@ def normalize (l : AList (fun _ : ℕ => Bool)) :
         · -- normalized
           have := ht₃ v
           have := he₃ v
-          ◾
+          simp_all -- used to be done here v4.7.0-rc1 issues
+          intro _
+          constructor <;> intro h <;> apply Option.some_ne_none _ <;>
+            rw [← AList.lookup_insert l]
+          · exact ht₃ _ h
+          · exact he₃ _ h
+          -- ◾-- aesop times out now
```

Comment:
@mattrobball let's try to carefully mark any regressions we make in fixes, e.g. with `FIXME nightly-testing`. We need to fix these properly before we can merge into `bump/v4.7.0` (and thence master), so it's important that incomplete patches are clearly marked as such.

The whole point of this file is to show off `aesop`, so writing out the proof is not necessarily better than a sorry.

Reply:
Yes, that is why it is marked with `v4.7.0-rc1` and why I mentioned this tag on Zulip.

Reply:
Ah, missed the messsage about v4.7.0-rc1, thanks.